src/components/hooks/dates.ts

```diff
@@ -10,6 +10,7 @@
 import React from 'react'
 import {formatDistance, Locale} from 'date-fns'
 import {
+  an,
```

Comment:
It looks like the `date-fns` package doesn't yet support the `an` locale, so I think this line should be removed.

https://github.com/date-fns/date-fns/tree/main/src/locale

Reply:
Thanks! I commited your change

Reply:
Nice! I think this line might need to be removed as well though; there is no entry for `ga` for instance, which is also not supported.

I didn't realise that you could suggest to remove a line like this, but apparently you can :)
```suggestion
```